web/pdf_page_view.js

```diff
@@ -452,11 +452,41 @@ class PDFPageView {
 
     let resultPromise = paintTask.promise.then(function() {
       return finishPaintTask(null).then(function () {
+        let pump;
```

Comment:
> fetch all of the textContent here before rendering the textLayer

I think we need `textContent` in text_layer_builder.js files, [here](https://github.com/mozilla/pdf.js/blob/master/web/text_layer_builder.js#L125), not calling `setTextContent` gives an error.